__tests__/jubilation-horizontal-bar/jubilation-horizontal-bar-test.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { defaultContext } from '../../src/helpers/context';
+import HorizontalBar from '../../src/jubilation-horizontal-bar';
+
+
+describe('JubilationHorizontalBarChart', () => {
+  it('should have a default data prop of []', () => {
+    expect(shallow(<HorizontalBar />).instance().props.data).toEqual([]);
+  });
+
+  it('should have a default gapWidth prop of 0.05', () => {
+    expect(shallow(<HorizontalBar />).instance().props.gapWidth).toBe(0.05);
+  });
+
+  it('should set a uuid attribute', () => {
+    const re = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/;
+    const uuid = shallow(<HorizontalBar />).instance().uuid;
+    expect(re.test(uuid)).toBe(true);
+  });
+
+  it('should add data as x data to the domain map', () => {
+    const { domainMap, uuid } = shallow(<HorizontalBar data={[1, 2]} />).instance();
+    expect(domainMap[uuid].x).toEqual([1, 2]);
+  });
+
+  it('should add dummy y data to the domain map', () => {
+    const { domainMap, uuid } = shallow(<HorizontalBar data={[1, 2]} />).instance();
+    expect(domainMap[uuid].y).toEqual([0, 0]);
+  });
+
+  it('shoud default to the base Jubilation context', () => {
+    const instance = shallow(<HorizontalBar />).instance();
+    expect(instance.context.JubilationContext).toEqual(defaultContext);
+  });
+
+  describe('ComponentWillRecieveProps', () => {
+    it('should add data as x data to the domain map', () => {
+      const wrapper = shallow(<HorizontalBar data={[1, 2]} />);
+      const { domainMap, uuid } = wrapper.setProps({ data: [3, 4] }).instance();
+      expect(domainMap[uuid].x).toEqual([3, 4]);
+    });
+
+    it('should add dummy y data to the domain map', () => {
+      const wrapper = shallow(<HorizontalBar data={[1, 2]} />);
+      const { domainMap, uuid } = wrapper.setProps({ data: [3, 4] }).instance();
+      expect(domainMap[uuid].y).toEqual([0, 0]);
+    });
+  });
+
+  describe('setXRange', () => {
+    it('should set the context xRange min to 5 if there are no labels', () => {
+      const instance = shallow(<HorizontalBar />).instance();
+      expect(instance.context.JubilationContext.xRange[0]).toBe(5);
+    });
+
+    it('should set the context xRange min to the longest label plus 5 if there are labels', () => {
+      const instance = shallow(<HorizontalBar labels={['test']} />).instance();
+      expect(instance.context.JubilationContext.xRange[0]).toBe(61);
+    });
+  });
+
+  describe('getHeight', () => {
+    it('should compute a height that takes gapWidth into account', () => {
+      const instance = shallow(<HorizontalBar data={[1]} />).instance();
+      expect(instance.getHeight()).toBe(90);
```

Comment:
This is `100-5-5` right?  i.e. 100 is `defaultContext.yRange[1]` and 5 is the default `gapWidth`?

It took me a second to hunt down where these numbers came from, might be nice to elucidate within the test.